internal/action/actions.go

```diff
@@ -1213,11 +1216,22 @@ func (h *BufPane) FindNext() bool {
 		InfoBar.Error(err)
 	}
 	if found {
+		if h.Buf.LastMatch[1] == match[0] && match[0] == match[1] {
```

Comment:
A comment like "skip empty match right after previous match" from #3566 would be also helpful here.

Reply:
fixed